### PR TITLE
Extract common part for currency to shared module

### PR DESCRIPTION
**Pull request by TheMMaciek**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -464,11 +464,30 @@ lazy val core = (project in file("modules/core"))
     )
   )
 
+lazy val currencyShared = (project in file("modules/currency-shared"))
+  .enablePlugins(AshScriptPlugin)
+  .enablePlugins(BuildInfoPlugin)
```
Comment (kpudlik) [inline build.sbt:469]: Do we need BuildInfo there?

**Review by kpudlik — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (kpudlik) [reply build.sbt:469]: Also add `currencyShared` to `root` so scalafix works

Comment (TheMMaciek) [reply build.sbt:469]: I guess we add buildInfo everywhere so I did the same here.

Comment (TheMMaciek) [reply build.sbt]: Not true actually. Removed the plugins that are not required at least right now.

**Review by kpudlik — APPROVED** [review]

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**